libimage/load.go

```diff
@@ -163,13 +187,34 @@ func (r *Runtime) loadMultiImageDockerArchive(ctx context.Context, ref types.Ima
 	var copiedImages []string
 	for _, list := range refLists {
 		for _, listRef := range list {
-			names, err := r.copyFromDockerArchiveReaderReference(ctx, reader, listRef, options)
+			images, err := r.copyFromDockerArchiveReaderReference(ctx, reader, listRef, options)
 			if err != nil {
 				return nil, err
 			}
-			copiedImages = append(copiedImages, names...)
+			names := []string{}
+			for _, img := range images {
+				if len(img.Names()) > 0 {
+					names = append(names, img.Names()...)
```

Comment:
All of the names, incl. names of all the deduplicated images which were not present in the archive?